app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate(params[:session][:password])
```

Comment:
`&.` -> Toán tử **Safe Navigation** này có tác dụng gì em?

Reply:
Toán tử &. có tác dụng là kiểm tra xem đối tượng gọi phương thức có phải là nil hay không. Nếu là nil thì nó sẽ trả về nil còn không thì nó sẽ gọi đến phương thức mà đối tượng đã gọi sau toán tử.
P/s: Câu này em trả lời ở pull request chapter 8 rồi ạ

Reply:
Sao chapter 10 lại có code chapter 8 em